bokeh/tests/test_properties.py

```diff
@@ -317,14 +317,28 @@ class IsDelegate(HasProps):
 
         class IncludesDelegateWithPrefix(HasProps):
             z = Include(IsDelegate, use_prefix=True)
-            z_y = String("world") # override the Include
+            z_y = Int(57) # override the Include
```

Comment:
Yes, Include is a macro, not a mixin. :-)  (If Include should have been a mixin, then there would be no reason to implement machinery for it at the metaclass level, because it would just be a regular class.)